README.md

```diff
@@ -123,3 +123,48 @@ jobs:
 ⚠️ Required GitHub permissions:
 
 - `actions: write`: Required to delete caches and artifacts.
+
+## `cache`
+
+Adaptive cache action that automatically chooses the appropriate caching backend based on repository visibility and ownership.
+
+Features:
+
+- Automatically uses GitHub Actions cache for public repositories
+- Uses SonarSource S3 cache for private/internal SonarSource repositories
+- Seamless API compatibility with standard GitHub Actions cache
+- Supports all standard cache inputs and outputs
+
+### Usage
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: SonarSource/ci-github-actions/cache@v1
+        with:
+          path: |
+            ~/.cache/pip
+            ~/.cache/maven
+          key: ${{ runner.os }}-cache-${{ hashFiles('**/requirements.txt', '**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-cache-
```

Comment:
Put the `cache-` prefix at first. That makes it easier to sort of the entries. And for consistency with the other actions.